manifests/global_defs.pp

```diff
@@ -121,6 +125,8 @@
   Optional[Integer[2, 3]] $vrrp_version              = undef,
   Optional[Integer[-1, 99]] $max_auto_priority       = undef,
   Boolean $dynamic_interfaces                        = false,
+  Optional[String] $vrrp_notify_fifo                 = undef,
```

Comment:
we could probably enforce an absolute path here as well?

Reply:
Yep, I will do that.